backend/sql/connection.js

```diff
@@ -1,14 +1,41 @@
 const mysql = require('mysql');
+const util = require('util');
 
 const db_config = {
+	//DEV database
 	connectionLimit: 100,
 	host: 'us-cdbr-iron-east-03.cleardb.net',
 	user: 'bd17b1984d3b71',
 	password: 'cc800052',
 	connectionTimeout: 10000,
-	database: 'heroku_f11d5f6ef0dc063'
+	database: 'heroku_f11d5f6ef0dc063',
+	multipleStatements: true
+
+	//PROD database 
+	/*
+	connectionLimit: 100,
+	host: 'us-cdbr-iron-east-03.cleardb.net',
+	user: 'bc3b1bec1b7226',
+	password: 'b6054e24',
+	connectionTimeout: 10000,
+	database: 'heroku_35d3c28bdd7d221'
+	*/
```

Comment:
this is just a nitpick, I know our app isnt like professional use ^^ but lets hide the credentials to our DB

Reply:
Hmm this would be tough to do and involve an additional encryption layer but I think I can come up with something, would need to add an additional task in the task list though as its definitly going to take me some time

Reply:
We can store the DATA base credentials in our local environment as Environment Variables, and then use sth like `process.env.ENV_VARIABLE` to get the credentials. But I think we can put this as low priority. As if we do this, everyone needs to store the credentials on their local environment.